### ISSUE TITLE
Search for "teams" is not working properly.

### ISSUE THREAD

**Issue by lovettchris** [microsoft/opensource-management-portal#266]
**Describe the bug**
I'm trying to create a new repo using https://repos.opensource.microsoft.com/orgs/microsoft/new and it requires a "admin team", so i created a new team here: https://github.com/orgs/microsoft/teams/ada-team but the find team dialog is not finding it:

When I search for "ada" it finds a million substring matches which also makes it impossible to find.  Can you fix the search box so it works on "whole word" or support double quotes searches like "ada team" that finds exact literal case insensitive match?

**Deployment environment**
This bug report is intended for issues relating to the project, and not a particular instance of the
management portal. If you are looking to file a bug report about the app itself, please instead
locate the links for support in the footer of every page of the app that you are using.

**To Reproduce**
See above.

**Expected behavior**
Search box needs to work better.

**Screenshots**

![image](https://user-images.githubusercontent.com/18707114/150616935-7ea1a679-2477-478e-8de5-af70da79f2e2.png)

**Additional context**

**Comment by jeffwilcox** [comment]
Thanks for the feedback, Chris.

Any chance you used the "refresh" button on the right at any time, or just the search experience?

**Comment by lovettchris** [comment]
Yes I was clicking refresh over and over, but I think I know what happened, the team was showing up here and it seems whatever teams are prepopulated there do not show up in the "Other teams..." list.

![image](https://user-images.githubusercontent.com/18707114/150861968-e38d5620-f10d-42c3-b026-a8d793a110e3.png)

But it does show up today, so is there just a very long delay in updating this list?
![image](https://user-images.githubusercontent.com/18707114/150862141-e1fca0be-028f-48a9-aae6-503747aeeecc.png)

**Comment by jeffwilcox** [comment]
Ah, yeah, that must be it. On the GitHub side it can take over 20m to get the list of teams updated in the Microsoft org when someone requests a refresh.

**Comment by lovettchris** [comment]
would help to perhaps say that somewhere on the site so I can not fret, grab a coffee and come back to it :-)

**Comment by jeffwilcox** [comment]
It's my understanding this issue in the Microsoft-internal instance has been fixed.
